4-graceful-sigint/main.go

```diff
@@ -13,10 +13,31 @@
 
 package main
 
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
 func main() {
 	// Create a process
 	proc := MockProcess{}
 
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT)
+
+	go func() {
+		sig := <-sigs
+		fmt.Println("\nReceived signal:", sig)
+		go proc.Stop() // спроба зупинити процес
+
+		// Чекаємо на другий сигнал
+		sig = <-sigs
```

Comment:
Значення `sig` не використовується, можемо видалити?
```suggestion
		<-sigs
```